lib/features/home_screen/widgets/contact_icon_widget.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../../../config/app_ui_config.dart';
+import '../../../theme/app_theme.dart';
+
+class ContactIconWidget extends StatelessWidget {
+  const ContactIconWidget({super.key, required this.url});
+
+  final String url;
+
+  @override
+  Widget build(BuildContext context) {
+    final Uri _url = Uri.parse('https://flutter.dev');
+
+    return GestureDetector(
+      onTap: () async {
+        if (!await launchUrl(_url)) {
+          throw Exception('Could not launch $_url');
+        }
+      },
+      child: Container(
+        decoration: BoxDecoration(
+          color: context.colorTheme.greyLight,
+          borderRadius: AppUiConfig.borderRadious,
+        ),
+        child: Icon(
+          Icons.ac_unit,
```

Comment:
For the default icon we can use some kind of web icon maybe?  But for hot days like one we have today maybe ac's better 😂😂

Reply:
I believe this is not even a default/fallback icon, but a **always go to** icon here.

Reply:
probably some regex detection of icon based on url like in topwr is an idea to consider - but I'm not sure if this will work well here, cause I don't know the context. PR states a "join to came text input", not contact icons